documentation/modules/kafkaconnectors/proc-deploying-kafkaconnector.adoc

```diff
@@ -43,12 +43,12 @@ spec:
 +
 [source,shell,subs="+quotes"]
 ----
-oc apply -f examples/connector/source-connector.yaml
+kubectl apply -f examples/connector/source-connector.yaml
 ----
 
 . Check that the resource was created:
 +
 [source,shell,subs="+quotes"]
 ----
-oc get all --selector strimzi.io/cluster: my-connect-cluster -o name
+kubectl get all --selector strimzi.io/cluster: my-connect-cluster -o name
```

Comment:
why these two changes in this PR. I am for avoiding stuff not related to the PR but opening a new one.

Reply:
I would revert this change here. It's addressed by PR #2422

Reply:
I'll avoid doing this in the future. 

I'll attempt to revert the change now that #2422 has been submitted.